python3/raygun4py/raygunmsgs.py

```diff
@@ -133,6 +133,7 @@ def get_details(self):
 
 class RaygunErrorMessage(object):
 
+    MAX_LOCAL_VAR_LENGTH = 4096 # TODO: parametrize this as a config option, like `transmit_environment_variables`.
```

Comment:
The payload of all the local variables was far too big, and some raygun errors were never making it.
```
>>> test.execute() # raises an exception:
send: b'POST /entries HTTP/1.1\r\nHost: api.raygun.io\r\nUser-Agent: raygun4py\r\nContent-Type: application/json\r\n
Content-Length: 2233969\r\n\r\n'
...truncated...
"occurredOn": "2020-11-13 15:27:38.265016"}'
[2020-11-13 15:27:38,708] {{raygunprovider.py:163}} ERROR - ('Connection aborted.', OSError("(32, 'EPIPE')",))
>>>
```
Notice that the content-length is: `2233969` about 2.2MB, most of it local variables.